common/middleware.py

```diff
@@ -39,7 +41,9 @@ def __call__(self, request):
                     else:
                         flattened_content[key] = resource
                 elif resource is None:
-                    flattened_content[list(content['data'].keys())[index]] = resource
+                    flattened_content[
+                        list(content['data'].keys())[index]
+                    ] = resource
 
```

Comment:
Although it makes clearer visually, I feel it looks awkward being on the left side of `=`.